string waypoint_name_destination
---
string waypoint_name